packages/cfa-template/template/backend/.env

```diff
@@ -0,0 +1,7 @@
+AES_KEY=Testing123456789Testing123456789
+AES_KEY_ID=
+MONGO_URI=mongodb://root:rootpassword@localhost:27017?retryWrites=true&w=majority
+REDIRECT_URI=http://localhost:3000/redirect
+HUBSPOT_CLIENT_ID=
+HUBSPOT_CLIENT_SECRET=
+HUBSPOT_SCOPE=oauth crm.objects.contacts.read crm.objects.companies.read crm.objects.companies.write crm.objects.contacts.write crm.schemas.deals.read crm.objects.deals.read
```

Comment:
This file should be on the `.gitignore`, right?

Reply:
It is, I included it in the template so that it just copies and writes it to the repo. (I had some finicky-ness to create-frigg-app that made it hard to generate the proper env files, so I just went with direct commit/add/write)